pageserver/src/metrics.rs

```diff
@@ -55,6 +55,9 @@ pub(crate) enum StorageTimeOperation {
 
     #[strum(serialize = "find gc cutoffs")]
     FindGcCutoffs,
+
+    #[strum(serialize = "layer flush wait upload")]
+    LayerFlushWaitUpload,
```

Comment:
Per-timeline histograms are our most expensive metrics in terms of impact on the monitoring infrastructure -- for the case we're looking at, my intuition is that we will mostly want to know "what percentage of walltime is the flush loop spending waiting for uploads?" -- i.e. look at the rate of the histogram's _sum

I don't mind merging the histogram if you find it useful in the near term, but I'd look to roll this back to a single counter later.

Reply:
I agree that percentage of walltime is enough and a gauge should be sufficient